### PR TITLE
Enhance/beautitfy page and navbars

### PR DESCRIPTION
**Pull request by ngerald51**
Nathan you may review the view pages that i have added and also templates

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -1,43 +1,54 @@
...
+          <%= f.password_field :current_password, autocomplete: "off"%>
+          <br/>
+
+          <div class="actions">
+            <%= f.submit "Update" , :class=>"ui primary submit labeled icon button" %>
+          </div>
+          <% end %>
+
+          <h3>Would you like to cancel your account?</h3>
+          <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete, :class=>"ui primary submit labeled icon button" %></p>
```
Comment (gohdaniel15) [inline app/views/devise/registrations/edit.html.erb:50]: Stick to a consistent hash declaration. Use `key: value` instead of `:key => value`

**Review by gohdaniel15 — COMMENTED** [review]
Code is fine, but the indentations and spacings are a mess. In ruby we use two single spaces for a tab. Set up your editor for that, edit the files in this commit (you can use a linter) and :rocket:

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -1,43 +1,54 @@
...
+    <div class="ui segment">
+      <div class="description">
+
+        <%= form_for(resource, as: resource_name, :class => 'ui form segment', url: registration_path(resource_name), html: { method: :put }) do |f| %>
+        <%= devise_error_messages! %>
+      </div>
+      <div class="ui divider"></div>
+      <div class='ui form segment'>
+
+
```
Comment (nndd91) [inline app/views/devise/registrations/edit.html.erb]: I think good practice is to not leave too many empty lines. Usually should keep to just one empty line

Comment (nndd91) [inline app/views/devise/registrations/edit.html.erb:18]: I think good practice is to not leave too many empty lines. Usually should keep to just one empty line

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -1,29 +1,40 @@
...
-<% end %>
-
-<%= render "devise/shared/links" %>
+  <div class="ui container">
+    <div class="ui segments">
+      <div class="ui segment orange inverted">
+        <h5 align="center">Registration for Twitter</h5>
+          <div class="ui segment">
+          <form class="ui form segment">
+        <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```
Comment (nndd91) [inline app/views/devise/registrations/new.html.erb:7]: try to indent this to follow proper styling guidelines.

File: app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -1,29 +1,40 @@
...
+
+        <div class="field">
+          <%= f.label :password_confirmation %><br />
+          <%= f.password_field :password_confirmation, autocomplete: "off" %>
+        </div>
+
+        <div class="actions">
+          <%= f.submit "Sign up", :class=>"ui primary submit labeled icon button" %>
+          
+
```
Comment (nndd91) [inline app/views/devise/registrations/new.html.erb:31]: as mentioned above, reduce the number of empty lines.

**Review by nndd91 — APPROVED** [review]
I see a few conflicts file when we merge. Lets discuss on the merging conflicts when you are free.

**Pull request merged**